catalog/core/catalog-core-urlresourcereader/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -30,7 +30,7 @@
         </set>
       </property>
   </bean>
-  
+
```

Comment:
I did have some changes in this file that were later removed. This resulted in some whitespace removal apparently but the white space shouldn't have been there in the first place so if you are ok with it I am going to leave it as is.